### ISSUE TITLE
Updates are anoyingly frequent, please add background updating

### ISSUE THREAD

**Issue by popcatalin81** [microsoft/AzureStorageExplorer#1565]
Hi,

Updates are way too frequent for Storage Explorer. I haven't opened the application in years without it prompting me for updates. It's true I'm an occasional user, but this is ridiculous. 

Please add background updating to the app.

Thanks

**Comment by jinglouMSFT** [comment]
Just curious, how often is an  acceptable update frequency to you?  We currently release updates approximately every two months, reduced from a monthly update frequency.  Thanks.

**Comment by popcatalin81** [comment]
Considering that I have several dozen products on my development machine, I'm bombarded with update requests almost daily. Almost every day I have to update either Visual Studio 2017, or 2019, or VS Code, or  Sql Management Studio, or some Teams, or Skype, or Several Addins, or Microsoft Storage explorer (Or Windows itself), or a dozen others, Weekly or Biweekly update schedules are a hell of an annoyance for the end-user. All these apps updating monthly or bi-monthly only start to bug you once a few days which is still annoying. Ideally, if you'd not have to deal with updates or only deal with them about once a month (As a whole, not per specific app) it would be really manageable.

**Comment by jinglouMSFT** [comment]
I understand what you are saying.  What I can help address is only on Storage Explorer however.  Do you feel comfortable having Storage Explorer auto-update completely without your permission? or you prefer a setting in Storage Explorer for you to opt-in for complete auto-updates after giving the permission once?

**Comment by popcatalin81** [comment]
@jinglouMSFT Yes, background, auto-updating would be really nice. I guess a setting would make the feature even nicer for users who want to opt-out, I don't necessarily want it for my use case.

**Comment by jinglouMSFT** [comment]
We usually release  every 1.5 to 2 months, including new work going into Blobs.  We can certainly consider adding an option to opt-in / opt-out.

**Comment by viblo** [comment]
Seems like this is the same issue as  #1058 and that the solution there would help here as well?
